js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -647,6 +648,63 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * In-App Purchases Implementation
+   */
+  /**
+   * This will retrieve the list of products deatils available for inapp-purchases from the play/app stores.
+   * @returns List of products for inapp-purchases
+   * @see {getProducts}
+   */
+  getProducts() {
+    return new Promise<Product[]>((resolve, reject) => {
+      return this.executor.exec(
+        'requestGetProducts',
+        null,
+        responseData => {
+          resolve(JSON.parse(responseData) as Product[]);
+        },
+        error => reject(parseMiniAppError(error))
+      );
+    });
+  }
+
+  /**
+   * This will requst for the inapp-purchase of a product with product id from the app/play store.
+   * @param id Product id of the product to be purchased.
+   * @returns Purchased product details and the transaction details of the purchase.
+   */
+  purchase(id: string) {
+    return new Promise<PurchasedProduct>((resolve, reject) => {
+      return this.executor.exec(
+        'requestPurchaseProduct',
+        { id },
+        responseData => {
+          resolve(JSON.parse(responseData) as PurchasedProduct);
+        },
+        error => reject(parseMiniAppError(error))
+      );
+    });
+  }
+
+  /**
+   * Retrieves the details of products purchased.
+   * @param ids List of purchased product ids.
+   * @returns list of product and transaction details for purchased products.
+   */
+  purchasedProducts(ids: string[]) {
+    return new Promise<PurchasedProduct[]>((resolve, reject) => {
+      return this.executor.exec(
+        'requestPurchasedProducts',
+        { ids },
+        responseData => {
+          resolve(JSON.parse(responseData) as PurchasedProduct[]);
+        },
+        error => reject(parseMiniAppError(error))
+      );
+    });
+  }
```

Comment:
We won't be supporting this interface for now. So we can remove this for now.

Reply:
This is now removed.